test/Microsoft.AspNet.Mvc.Core.Test/PhysicalFileResultTest.cs

```diff
@@ -131,19 +132,18 @@ public void Constructor_SetsFileName()
         [InlineData("..\\TestFiles\\SubFolder/SubFolderTestFile.txt")]
         [InlineData("~/SubFolder/SubFolderTestFile.txt")]
         [InlineData("~/SubFolder\\SubFolderTestFile.txt")]
-        public async Task ExecuteAsync_ThrowsFileNotFound_ForNonRootedPaths(string path)
+        public async Task ExecuteAsync_ThrowsNotSupported_ForNonRootedPaths(string path)
         {
             // Arrange
             var result = new TestPhysicalFileResult(path, "text/plain");
             var context = new ActionContext(new DefaultHttpContext(), new RouteData(), new ActionDescriptor());
-            var expectedMessage = "Could not find file: " + path;
+            var expectedMessage = string.Format("Given path '{0}' was not rooted. Paths must be rooted.", path);
```

Comment:
`var expectedMessage = $"Given path '{path}' was not rooted. Paths must be rooted.";`

Also should the message say `Path {path} was not rooted.` instead? The current error message seems a bit atypical.